internal/agent/install.go

```diff
@@ -250,7 +250,28 @@ func RunInstall(c *config.Config) error {
 	// Set our cloud-init to the file we just created
 	installSpec.CloudInit = append(installSpec.CloudInit, f.Name())
 	// Get the source of the installation if we are overriding it
-	if c.Install.Image != "" {
+	if sourceImg != "" {
+		imgSource, err := v1.NewSrcFromURI(sourceImg)
+		if err != nil {
+			return err
+		}
+		installSpec.Active.Source = imgSource
+
+		// TODO: Why only setting active source above? What about size?
```

Comment:
on install, only the active is used as a source, as it makes no sense having separated sources for all 3 during an install. There is only 1 source for a clean install and its used for all three images (active,passive,recovery)

hence, no need to set anything else. You can see this on the `NewInstallSpec`